src/Models/Constants/dtdlInterfaces.ts

```diff
@@ -122,8 +122,8 @@ export interface DtdlArray {
 export interface DtdlMapKey {
     /** The "programming" name of the map's value. The name may only contain the characters a-z, A-Z, 0-9, and underscore, and must match this regular expression `^[a-zA-Z](?:[a-zA-Z0-9_]*[a-zA-Z0-9])?$.` */
     name: string;
-    /** The data type of the map's key */
-    schema: DTDLSchema;
+    /** The data type of the map's key. Always `string` */
+    schema: 'string';
```

Comment:
Why do we need 'string' literally defined here?

Reply:
becuase it has this property called `Schema` but the value is always defined as 'string'. I keep it here cause we do type checking on the `schema` field a lot. And everything in DTDL has a schema, it's just that sometimes, that field is a fixed constant. It's the same way with the @type field.